Library/Homebrew/rubocops/caveats_cop.rb

```diff
@@ -0,0 +1,19 @@
+require_relative "./extend/formula_cop"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      class Caveats < FormulaCop
+        def audit_formula(_node, _class_node, _parent_class_node, body_node)
+          caveats_node = find_method_def(body_node, :caveats)
+          return if caveats_node.nil?
+          caveats_string_nodes = find_strings(caveats_node)
+          caveats_string_nodes.each do |n|
```

Comment:
Would be good if we could get all the caveats strings nodes as a single method call?